src/accessors.jl

```diff
@@ -49,6 +49,9 @@ get_loads_per_bus(system::System) = system.loads_per_bus
 
 "Returns the power transfer distribution factor of the system."
 get_ptdf(system::System) = system.ptdf
+"Returns the power transfer distribution factor of the system. Calculates it if missing from system"
+ptdf(system::System) = coalesce(get_ptdf(system), compute_ptdf(system))
```

Comment:
Only if we use both in the same scope. I just used the names from the original issue but if you can think of a better one I'd be happy to change it.

Reply:
What about `retrieve_ptdf`?
- `get_ptdf` is the plain accessor
- `compute_ptdf` is the "compute no matter what" option
- `retrieve_ptdf` is the "compute if not already present" option.  
 
If you don't like it we can stick with `ptdf`, because I don't think we use it as a variable name in a scope where it would be problematic.

Reply:
`retrieve_ptdf` seems like a fine name and makes it easier to change over our internal code. I'll make the change.